solutions/devicesimulation/armtemplate/template.json

```diff
@@ -272,411 +278,445 @@
-                        }
-                    ]
-                }
-            },
-            "resources": [
-                {
-                    "type": "sourcecontrols",
-                    "name": "web",
-                    "apiVersion": "2015-08-01",
-                    "properties": {
-                        "RepoUrl": "[variables('webAppRepoURL')]",
-                        "branch": "[variables('webAppRepoBranch')]",
-                        "IsManualIntegration": true
-                    },
-                    "dependsOn": [
-                        "[resourceId('Microsoft.Web/Sites', parameters('azureWebsiteName'))]"
-                    ]
-                }
-            ],
-            "tags": {
-                "IotSuiteType": "[parameters('solutionType')]"
+              "protocol": "TCP",
+              "sourcePortRange": "*",
+              "destinationPortRange": "22",
+              "sourceAddressPrefix": "*",
+              "destinationAddressPrefix": "*",
+              "access": "Deny",
+              "priority": 101,
+              "direction": "Inbound"
             }
-        },
-        {
-            "comments": "Security rules used for the VM network interface",
-            "type": "Microsoft.Network/networkSecurityGroups",
-            "name": "[variables('networkSecurityGroupName')]",
-            "apiVersion": "[variables('networkApiVersion')]",
-            "location": "[variables('location')]",
-            "tags": {
-                "IotSuiteType": "[parameters('solutionType')]"
-            },
+          },
+          {
+            "name": "SSH_alt",
             "properties": {
-                "securityRules": [
-                    {
-                        "name": "HTTPS",
-                        "properties": {
-                            "protocol": "TCP",
-                            "sourcePortRange": "*",
-                            "destinationPortRange": "443",
-                            "sourceAddressPrefix": "*",
-                            "destinationAddressPrefix": "*",
-                            "access": "Allow",
-                            "priority": 100,
-                            "direction": "Inbound"
-                        }
-                    },
-                    {
-                        "name": "SSH",
-                        "properties": {
-                            "protocol": "TCP",
-                            "sourcePortRange": "*",
-                            "destinationPortRange": "22",
-                            "sourceAddressPrefix": "*",
-                            "destinationAddressPrefix": "*",
-                            "access": "Deny",
-                            "priority": 101,
-                            "direction": "Inbound"
-                        }
-                    }
-                ]
+              "protocol": "TCP",
+              "sourcePortRange": "*",
+              "destinationPortRange": "10022",
```

Comment:
remove port 10022 (search globally) - it was used only for PoC

Reply:
done